cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala

```diff
@@ -601,9 +602,243 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
+            p("Lists all privileges for role 'deniedBoostedProcedureExecutor2'")
+            resultTable()
+          }
+
+          p("Example 3: Grant `EXECUTE BOOSTED PROCEDURE` and deny `EXECUTE BOOSTED PROCEDURE`")
+          query("GRANT EXECUTE BOOSTED PROCEDURE * ON DBMS TO deniedBoostedProcedureExecutor3", ResultAssertions((r) => {
+            assertStats(r, systemUpdates = 1)
+          })) {
+            statsOnlyResultTable()
+          }
+          query("DENY EXECUTE BOOSTED PROCEDURE db.labels ON DBMS TO deniedBoostedProcedureExecutor3", ResultAssertions((r) => {
+            assertStats(r, systemUpdates = 1)
+          })) {
+            statsOnlyResultTable()
+          }
+          p(
+            """The resulting role should have privileges that allow executing all procedures with elevated privileges
+              |except `db.labels` which is not allowed to execute at all:""".stripMargin)
+          query("SHOW ROLE deniedBoostedProcedureExecutor3 PRIVILEGES", assertPrivilegeShown(Seq(Map()))) {
+            p("Lists all privileges for role 'deniedBoostedProcedureExecutor3'")
+            resultTable()
+          }
+
+          p("Example 4: Grant `EXECUTE PROCEDURE` and `EXECUTE BOOSTED PROCEDURE` and deny `EXECUTE BOOSTED PROCEDURE`")
+          query("GRANT EXECUTE PROCEDURE db.labels ON DBMS TO deniedBoostedProcedureExecutor4", ResultAssertions((r) => {
+            assertStats(r, systemUpdates = 1)
+          })) {
+            statsOnlyResultTable()
+          }
+          query("GRANT EXECUTE BOOSTED PROCEDURE * ON DBMS TO deniedBoostedProcedureExecutor4", ResultAssertions((r) => {
+            assertStats(r, systemUpdates = 1)
+          })) {
+            statsOnlyResultTable()
+          }
+          query("DENY EXECUTE BOOSTED PROCEDURE db.labels ON DBMS TO deniedBoostedProcedureExecutor4", ResultAssertions((r) => {
+            assertStats(r, systemUpdates = 1)
+          })) {
+            statsOnlyResultTable()
+          }
+          p(
+            """The resulting role should have privileges that allow executing all procedures with elevated privileges
+              |except `db.labels` which is only allowed to execute using the users own privileges:""".stripMargin)
+          query("SHOW ROLE deniedBoostedProcedureExecutor4 PRIVILEGES", assertPrivilegeShown(Seq(Map()))) {
+            p("Lists all privileges for role 'deniedBoostedProcedureExecutor4'")
+            resultTable()
+          }
+        }
+
+        section("The `EXECUTE ADMIN PROCEDURES` privilege", "admin-execute-procedure-subsection", "enterprise-edition") {
+          p(
+            """The ability to execute admin procedures (annotated with `@Admin`) can be granted via the `EXECUTE ADMIN PROCEDURES` privilege.
+              |This privilege is equivalent with granting <<boosted-execute-procedure-subsection, the `EXECUTE BOOSTED PROCEDURE` privilege>> on each of the admin procedures.
```

Comment:
Should we say something about if a new admin procedure is added, that it will be executable?

Reply:
Probably a good idea yes